Resources/public/scss/modules/sub-items-list/_grid.view.scss

```diff
@@ -5,8 +5,8 @@
     padding-top: calculateRem(8px);
 
     .c-grid-view-item {
-        flex: 1 1 calc(100% / 6);
+        flex: 1 1 calc(100% / 5);
         max-width: calculateRem(166px);
-        margin-right: calculateRem(8px);
+        margin-right: calculateRem(35.75px);
```

Comment:
This value is very big and strange. Could you explain why it has to be `35.75px`?